sherpa/astro/ui/tests/test_astro_ui_plot.py

```diff
@@ -92,10 +111,19 @@ def example_pha_data():
     return d
 
 
-def example_pha_with_bkg_data():
+def example_pha_with_bkg_data(direct=True):
     """Create an example data set with background
 
     There is no response for the background.
+
+    Parameters
+    ----------
+    direct : bool, optional
+        If True then the background is added to the source
+        and only the source is returned. If False then the
+        return is (source, bgnd) and the backgrouund is not
```

Comment:
typo- backgrouund -> nackground

Reply:
As I had to rebase because of a conflict I fixed this in the original commit: https://github.com/sherpa/sherpa/pull/882/commits/74c53ef9c2e36c9a553d0b41c053f0ecc124c472#diff-a6ca41fadd396de931aed280f1fbd433R105

Reply:
Oops - I'm going to have to rebase again, so the commit will change, but the fix has been made.